data_management/opensearch/tests/test_lambda_handler.py

```diff
@@ -0,0 +1,128 @@
+import json
```

Comment:
this is a bit of a question but would we want to extend the unit_test workflow to include running tests for additional modules like this?

Reply:
great point, yes! I will adjust it now